scripts/update-lockfile.sh

```diff
@@ -12,9 +12,7 @@ make_ignored() {
   fi
 }
 
-TO_IGNORE=$(make_ignored $1)
-
```

Comment:
this is still wrong

Reply:
I doubt that this worked at any time before... Even the original master does not work with `update-lockfile python" Will implement it in lua

Reply:
should work now